lucene_qat_wrapper/lucene_with_qat.diff

```diff
@@ -0,0 +1,588 @@
+diff --git a/lucene/build.xml b/lucene/build.xml
```

Comment:
This file is intended for Lucence side to apply. We should provide version info to it. You can do it by creating related version number as folder name like what we did for Parquet/Hive.